app/src/main/java/ch/sdp/vibester/activity/AuthenticationActivity.kt

```diff
@@ -99,7 +99,7 @@ class AuthenticationActivity : AppCompatActivity() {
      * Listener bound to the red return button in the Authentication activity.
      */
     fun returnToMainListener(view: View) {
-        IntentSwitcher.switchBackToWelcome(this)
+        IntentSwitcher.switch(this, WelcomeActivity::class.java, null)
```

Comment:
Maybe a small comment to expain why we pass null here?

Reply:
I've removed the null, not it is by default